tests/unit/db/redshift/test_sql.py

```diff
@@ -12,11 +12,5 @@ def test_schema_sql_from_admin_views_not_installed(self,
         mock_table = Mock(name='table')
         mock_schema = Mock(name='schema')
         mock_db.execute.side_effect = sqlalchemy.exc.ProgrammingError('statement', {}, {})
-        out = schema_sql_from_admin_views(mock_schema, mock_table, mock_db)
-        self.assertIsNone(out)
-        mock_logger.debug.assert_called_with('Error while generating SQL', exc_info=True)
-        mock_logger.warning.\
-            assert_called_with("To be able to save SQL to a records directory, "
-                               "please install and grant access to 'admin.v_generate_tbl_ddl' "
-                               "from https://github.com/awslabs/amazon-redshift-utils/"
-                               "blob/master/src/AdminViews/v_generate_tbl_ddl.sql")
+        with self.assertRaises(AttributeError):
+            schema_sql_from_admin_views(mock_schema, mock_table, mock_db)
```

Comment:
that logging error seems useful.. does it still get logged or is the new behavior simply an attribute error?

Reply:
This now calls correctly by using MagicMocks